app/views/course/assessment/submissions/_worksheet.html.slim

```diff
@@ -0,0 +1,59 @@
+= simple_form_for [current_course, @assessment, @submission] do |f|
+  = f.error_notification
+  = f.simple_fields_for :answers do |base_answer_form|
+    = content_tag_for(:div, base_answer_form.object) do
+      = render partial: 'course/assessment/answer/answer', object: base_answer_form.object,
+               locals: { base_answer_form: base_answer_form }
+
+  - unless @submission.attempting?
+    div.panel.panel-default
+      div.panel-heading = t('.statistics')
+      div.panel-body.statistics
+        table.table.table-striped
+          tbody
+            tr
+              th = t('.student')
+              td = display_course_user(@submission.course_user)
+            tr
+              th = t('.status')
+              td = @submission.workflow_state
+            tr
+              th = @submission.class.human_attribute_name(:grade)
+              td = @submission.grade
+            tr
+              th = @submission.experience_points_record.class.human_attribute_name(:points_awarded)
+              td = @submission.points_awarded
+            tr
+              th = t('.due_at')
+              td = format_datetime(@assessment.end_at) if @assessment.end_at
+            tr
+              th = @submission.class.human_attribute_name(:submitted_at)
+              td = format_datetime(@submission.submitted_at)
+            tr
+              th = @submission.class.human_attribute_name(:grader)
+              td = @submission.graders.uniq
+            tr
+              th = @submission.class.human_attribute_name(:graded_at)
+              td = format_datetime(@submission.graded_at) if @submission.graded_at
+
+        - if @submission.answers.length > 1
```

Comment:
I don’t know where to place it for trainings either. Factor it out to a partial first, then.